src/pages/project/Project.jsx

```diff
@@ -1,12 +0,0 @@
-import React from "react";
```

Comment:
This component is unnecessary and we can remove it because it is not doing anything